pkg/apis/kops/validation/validation.go

```diff
@@ -1517,7 +1517,7 @@ func validateNodeLocalDNS(spec *kops.ClusterSpec, fldpath *field.Path) field.Err
 }
 
 func validateClusterAutoscaler(cluster *kops.Cluster, spec *kops.ClusterAutoscalerConfig, fldPath *field.Path) (allErrs field.ErrorList) {
-	allErrs = append(allErrs, IsValidValue(fldPath.Child("expander"), spec.Expander, []string{"least-waste", "random", "most-pods"})...)
+	allErrs = append(allErrs, IsValidValue(fldPath.Child("expander"), spec.Expander, []string{"least-waste", "random", "most-pods", "price", "priority"})...)
```

Comment:
Is price actually supported on AWS? If it is only for certain cloud vendors, we should validate that.

Reply:
Good point, I could add validation and docs for that.

Reply:
Added validation and docs. My knowledge of Go is basically zero so please let me know if I've done anything dumb.